### PR TITLE
Feature/display user history

### PR DESCRIPTION
**Pull request by elizhahn**
# Is this a fix or a feature?
- feature
# What is the change?
- This adds functionality to the application and shows a user their history on a dashboard
- This takes care of user stories: 
    - As a customer I should see any room bookings I have made (past or present/upcoming)
    - As a customer I should see the total amount I have spent on rooms
# What does it fix?
# Where should the reviewer start?
- index.js: lines: 14-59
# How should this be tested?
- clone the repository into a testing directory
- fetch the feature/display-user-history branch
- run npm start in the repo
- run npm start in the api
- Visit: http://localhost:8080/
- You should see a dashboard displaying total stays, spent, and stay history
- In addition the user's name should be displayed in a welcome message

# Screenshots (if appropriate):
Screen Shot 2021-03-06 at 1 53 25 PM

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,9 +3,63 @@
...
+const totalStays = document.getElementById('numberStays');
+const totalSpent = document.getElementById('totalSpent');
+const userHistory = document.getElementById('userHistory');
+const userName = document.getElementById('userName'); 
+let currentUser;
+let hotel;  
+
+const apiData = [fetch('http://localhost:3001/api/v1/customers/1'), fetch('http://localhost:3001/api/v1/rooms'), fetch('http://localhost:3001/api/v1/bookings')]
+
+Promise.all(apiData)
```
Comment (alexthompson207) [inline src/index.js:23]: I'm not sure what the best practice is for when to all a GET request, here it seems like it's being called on page load? Maybe move this to a function? Not sure though..

**Review by alexthompson207 — COMMENTED** [review]
Your overall website looks amazing! This is an awesome start! I left some feedback about your GET request, and how it seems to be just invoked right away. Is that what you want? 

Also the other thing I noticed, not a big deal, but a user currently can select a date from the past..

**Comment by alexthompson207** [comment]
Awesome work! This looks so good, you're off to a great start! I left a few comments above.

**Comment by elizhahn** [comment]
Thanks for the feedback, 
- Re: calendar: I have it set up that way for now because the data is dated for the past, so I wanted to be able to select past dates to see if the data shows up before I get to the POST part of the application
- Re: GET: I'm not sure either, I guess I was thinking, the sooner it can fire off and get the data the better, so I just have it happening automatically, I'm not sure the implications for when POST starts to get implemented. I'll def keep it in mind

**Pull request merged**